src/test/resources/application-testad.yml

```diff
@@ -0,0 +1,30 @@
+spring:
+  ldap:
+    embedded:
+      base-dn: DC=example,DC=com
+      credential:
+        username: CN=John Smith,CN=Users,DC=example,DC=com
+        password: password
+      ldif: classpath:ad-users.ldif
+      port: 9389
+      validation:
+        enabled: true
+authenticator:
```

Comment:
Why this method config is put separately while all others are in `src/test/resources/application.yml`? I would keep things consistent